package.json

```diff
@@ -81,7 +81,8 @@
     "source-map-explorer": "^2.5.2",
     "start-server-and-test": "^1.11.0",
     "typescript": "^4.1.2",
-    "web3": "^1.3.5"
+    "web3": "^1.3.5",
+    "d3": "^7.0.0"
```

Comment:
What is d3?

Reply:
d3 library  is for price range charts
![Screenshot from 2022-07-30 17-35-09](https://user-images.githubusercontent.com/20106622/181911705-af2930cf-cf15-4ea1-b6e0-1233a203a969.png)